src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Synthesized.cs

```diff
@@ -673,7 +664,7 @@ public static void Main()
     {
     }
 }";
-            var options = new CSharpCompilationOptions(outputKind, optimizationLevel: optimizationLevel);
+            var options = TestOptions.DebugDll.WithOutputKind(outputKind).WithOptimizationLevel(optimizationLevel);
```

Comment:
I feel it's more appropriate to use `CreateTestOptions` method that is now introduced in `TestOptions` class. But this will require a higher access modifier than private for that new method. The current approach just creates an arbitrary output kind and optimization level (DebugDll), then overwrite it with call chain to `.WithOutputKind(..).WithOptimizationLevel(..)`.

Reply:
I agree, it will be good to change accessibility of that method to internal (which should make it accessible from the tests).